spec/order_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'order'
+
+describe 'Order' do
+
+  subject(:order) { Order.new }
+  let(:menu) { double :menu }
+
+  describe '#initialize' do
+    it 'should have an empty order list' do
+      expect(order.list).to be_empty
+    end
+
+    it 'should have a menu' do
+      expect(order.menu).not_to be(nil)
+    end
+
+  end
+
+  describe '#read_menu' do
+    it 'displays the menu' do
+      expect{order.read_menu}.to output.to_stdout
+    end
+  end
+
+  describe '#choose' do
+    it 'allows customer to choose items and add to list' do
+       order.choose("Margherita")
+       expect(order.list).to include("Margherita" => 6)
+     end
```

Comment:
Lint/BlockAlignment: `end` at 29, 5 is not aligned with `it 'allows customer to choose items and add to list' do` at 26, 4.